roofit/multiprocess/inc/RooFit/MultiProcess/Config.h

```diff
@@ -13,6 +13,9 @@
 #ifndef ROOT_ROOFIT_MultiProcess_Config
 #define ROOT_ROOFIT_MultiProcess_Config
 
+#include "RooFit/MultiProcess/types.h"
```

Comment:
You are including something from `res` in a header file that is in `inc` now.

This is now allowed, because the headers from `inc` get installed to the user, but the ones from `res` don't. So now it's not possible to include `Config.h` anymore after the install.

The solution is to either move `types.h` to `inc`, or you move `Config.h` to `res`.

I would prefer the latter, because `RooFit/MultiProcess/Config.h` is not used in other public headers itself. But if you think public access to `Config.h` is necessary (even if only for your convenience as a developer), then feel free to move `types.h` instead!

Reply:
Yes, `Config` is the public interface to behind-the-scenes options to MultiProcess-using classes. It would not be so nice if we had to expose everything through the classes that use the options, that would be a lot of code duplication and needless redirection. I think the public `Config` (at least for now) still makes sense.

I will move `types.h` to `inc` then. It is quite light anyway, just used for slightly more meaningful types, even though in reality they are all just `std::size_t`s.

Reply:
Alright!